forge-ai/src/main/java/forge/ai/CreatureEvaluator.java

```diff
@@ -119,6 +119,11 @@ else if (c.hasKeyword(Keyword.WITHER)) {
             value += addValue(c.getKeywordMagnitude(Keyword.TOXIC) * 5, "toxic");
             value += addValue(c.getKeywordMagnitude(Keyword.AFFLICT) * 5, "afflict");
             value += addValue(c.getKeywordMagnitude(Keyword.RAMPAGE), "rampage");
+
+            if (c.hasTriggerContaining("Whenever this creature deals combat damage") ||
```

Comment:
Imo would be better to check if card has a Trigger depending on the Trigger Mode (for DamageDone) or something like that

Reply:
I've switched to checking for trigger types.